src/foam/blob/BlobJava.js

```diff
@@ -190,21 +190,21 @@ foam.CLASS({
     'org.apache.commons.io.IOUtils',
     'org.apache.commons.codec.binary.Hex',
     'java.io.File',
-    'java.io.FileOutputStream'
+    'java.io.FileOutputStream',
+    'foam.nanos.fs.Storage'
   ],
 
   constants: [
     {
-      name: 'BUFFER_SIZE',
       value: 4096,
+      name: 'BUFFER_SIZE',
```

Comment:
Why switch the order? I think it's more conventional to have the name before the value.